integration/Makefile.mk

```diff
@@ -29,7 +29,7 @@
 #  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 
-EXTRA_DIST += test_integration/test_tutorial_base.py \
-              test_integration/test_tutorial_base.sh \
+EXTRA_DIST += integration/__init__.py \
```

Comment:
This is git being confused by rename.  test_tutorial_base.mk is introduced as a "new file" below